pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/dao/ddp/instance/DDPInstanceDao.java

```diff
@@ -42,18 +42,21 @@ public class DDPInstanceDao implements Dao<DDPInstanceDto> {
     public static final String STUDY_PRE_FILTER = "study_pre_filter";
     public static final String QUERY_ITEMS = "query_items";
     public static final String RESEARCH_PROJECT = "research_project";
+    public static final String MERCURY_ORDER_CREATOR = "mercury_order_creator";
 
     private static final String SQL_INSERT_DDP_INSTANCE =
             "INSERT INTO ddp_instance SET  instance_name = ?,  study_guid = ?, display_name = ?, base_url = ?,"
                     + "is_active = ?, bsp_group = ?, bsp_collection = ?, bsp_organism = ?, collaborator_id_prefix = ?,"
                     + "reminder_notification_wks = ?, mr_attention_flag_d = ?, tissue_attention_flag_d = ?, auth0_token = ?,"
                     + "notification_recipients = ?, migrated_ddp = ?, billing_reference = ?, es_participant_index = ?,"
-                    + "es_activity_definition_index = ?, es_users_index = ?, study_pre_filter = ?, research_project = ?";
+                    + "es_activity_definition_index = ?, es_users_index = ?, study_pre_filter = ?, research_project = ?, "
+                    + "mercury_order_creator= ? ";
 
     private static final String SQL_DELETE_DDP_INSTANCE = "DELETE FROM ddp_instance WHERE ddp_instance_id = ?";
     private static final String SQL_SELECT_INSTANCE_WITH_ROLE =
             "SELECT ddp_instance_id, instance_name, base_url, collaborator_id_prefix, migrated_ddp, billing_reference, "
-                    + "es_participant_index, es_activity_definition_index, es_users_index, study_pre_filter, research_project, "
+                    +
```

Comment:
can we not do just `+` in a line… move your `mercury….` into the same line as `(select`, please